generators/app/tasks/installDependencies.js

```diff
@@ -31,7 +31,8 @@ const DEV_DEPENDENCIES = [
   'prettier',
   'prettier-eslint',
   'react-scripts',
-  'wolox-react-scripts'
+  'wolox-react-scripts',
+  '@storybook/react'
```

Comment:
Shouldn't this be listed as optional ? What do you think guys ?

Reply:
The way I pictured this is that, after you generate a project, you can run `npm run storybook` and already have examples of all default components the bootstrap gives you. And eventually, add each component in the storybook as you code.